doc/jsonrpc.md

```diff
@@ -3844,6 +3844,7 @@ no_srq                      | Optional | boolean | Disable shared receive queue
 c2h_success                 | Optional | boolean | Disable C2H success optimization (TCP only)
 dif_insert_or_strip         | Optional | boolean | Enable DIF insert for write I/O and DIF strip for read I/O DIF (TCP only)
 sock_priority               | Optional | number  | The socket priority of the connection owned by this transport (TCP only)
+wr_batching                 | Optional | boolean | Disable work requests batching (RDMA only)
```

Comment:
For backward compatibility, it is better to preserve value `0` or `false` for old behaviour. In this case WR batching was enabled by default before your changes, so I would suggest renaming this parameter to `no_wr_batching` and value `false` will enable it by default.